config/default/config_ignore.settings.yml

```diff
@@ -13,6 +13,7 @@ ignored_config_entities:
   - 'config_split.config_split.scholarship:status'
   - 'config_split.config_split.sitenow_migrate:status'
   - 'config_split.config_split.sitenow_v2:status'
+  - 'config_split.config_split.*:status'
```

Comment:
I think this would ignore site split statuses which would be undesired. Maybe we should prefix feature splits with feature_ or something.

Reply:
Good call.

Reply:
Is this causing the issue in the conflicting file however?

Reply:
I changed this and fixed the conflict.